openfisca_web_api_preview/openAPI.yml

```diff
@@ -0,0 +1,253 @@
+swagger: "2.0"
+info:
+  title: "{CONTRY_PACKAGE_NAME} Web API"
+  description: "The OpenFisca Web API lets you get up-to-date information and formulas pertaining to the {CONTRY_PACKAGE_NAME} legislation e.g. minimum wage, state scholasrship, social benefits, income taxes…"
```

Comment:
I don't think we should give these examples here: the country package name might not provide these elements. If we don't know how to explain better, then at least we should make it clearer that these are illustrative and might not be provided.